docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+version: '3'
```

Comment:
Version is deprecated

Reply:
sorry missed. Now updated the PR comments detailing all the changes.